tests/test_slots.py

```diff
@@ -660,10 +660,12 @@ def test_no_getstate_setstate_for_dict_classes(self):
         As long as getstate_setstate is None, nothing is done to dict
         classes.
         """
-        i = C1(1, 2)
-
-        assert None is getattr(i, "__getstate__", None)
-        assert None is getattr(i, "__setstate__", None)
+        assert getattr(object, "__getstate__", None) == getattr(
```

Comment:
Shouldn't `is` work here and be more correct too? I seem to remember that it didn't work on Python 2, but that shouldn't be an issue anymore.